src/App.tsx

```diff
@@ -1,25 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const total = 42;
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
```

Comment:
The 'perPage' state is initialized with a literal value. Consider defining this as a constant outside the component to improve maintainability and provide clarity on what this 'magic number' represents.

Reply:
Ok